opslevel/common.go

```diff
@@ -397,3 +397,12 @@ func reconcileStringArray(current []string, desired []string, add reconcileStrin
 	}
 	return nil
 }
+
+// Handles conversion from Terraform's interface{} struct to OpsLevel's JSON struct
+func convertHeadersMap(headers interface{}) opslevel.JSON {
+	output := opslevel.JSON{}
+	for k, v := range headers.(map[string]interface{}) {
```

Comment:
I think this needs to be

```suggestion
	for k, v := range headers.(map[string]string) {
```

because thats the type that `opslevel.JSON` is